### ISSUE TITLE
Rotate/revoke issuer.jwk throws error when origin auto-registers at registry

### ISSUE THREAD

**Issue by haoming29** [PelicanPlatform/pelican#383]
Steps to reproduce:
* If you have `issuer.jwk` under `/etc/pelican`, and have an origin prefix registered at registry, say `/origin1`
* `rm issuer.jwk`
* Run `director`, `registry`, and `origin`
* You would expect the following message:

```console
Error: Namespace /origin1 already registered under a different key
```

I'm not sure what's the fix for this, which can initiate to some security policy discussions. @bbockelm What's your thoughts on this?

**Comment by bbockelm** [comment]
That's the expected / desired behavior: if you try to register with a different key, it should be an error (otherwise someone could pretty trivially steal your namespace).

Separately, I would like to have a mechanism to rotate (add / delete) keys that are registered -- for example, if there are multiple origins under a single part of the namespace -- but you still need to have access to the old key to rotate.

**Comment by haoming29** [comment]
Closed with #561 opened
